beacon_node/http_api/tests/tests.rs

```diff
@@ -2616,12 +2616,18 @@ impl ApiTester {
     }
 
     pub async fn test_get_config_spec(self) -> Self {
-        let result = self
-            .client
-            .get_config_spec::<ConfigAndPresetFulu>()
-            .await
-            .map(|res| ConfigAndPreset::Fulu(res.data))
-            .unwrap();
+        let result = if self.chain.spec.fulu_fork_epoch.is_some() {
```

Comment:
I think this would always return true today, as we have `FULU_FORK_EPOCH` set to `FAR_FUTURE_EPOCH` on the mainnet spec

```suggestion
        let result = if self.chain.spec.is_fulu_scheduled() {
```

Reply:
Huh, I thought far future got converted to None? I'm afk but otherwise what is the point of the option?

Reply:
Yeah it's the same thing, see `deserialize_fork_epoch`.

However I don't mind using the safer `is_fulu_scheduled` method.